tests/designs/viterbi_decoder_axi4s/Makefile

```diff
@@ -65,8 +65,10 @@ VHDL_SOURCES = $(SRC_BASE)/packages/pkg_helper.vhd \
 	       $(SRC_BASE)/src/recursion.vhd \
 	       $(SRC_BASE)/src/dec_viterbi.vhd
 
-ifeq ($(SIM),$(filter $(SIM),ius xcelium))
-    SIM_ARGS += -v93
+ifneq (,$(SIM))
+    ifeq ($(SIM),$(filter $(SIM),ius xcelium))
+        SIM_ARGS += -v93
+    endif
 endif
```

Comment:
Simpler as:
```
ifneq ($(filter $(SIM),ius xcelium),)
    SIM_ARGS += -v93
endif
```

Reply:
That's fair, @eric-wieser. Let's see where we're actually putting this fix, and I'll make the changes.

Reply:
Focused this PR on Icarus argument passing